src/components/deposit/LotSizeSelector.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react'
+import classNames from 'classnames'
+
+const LotSizeOption = ({ lotSize, onClick, selected }) => {
+  const handleClick = () => {
+    onClick(lotSize)
+  }
+  return (
+    <li className={classNames('lot-size-option', { selected })}
+      onClick={handleClick}>
+        <input type="radio" id={lotSize} name="lot-size" value={lotSize} 
+          checked={selected} readOnly />
+        <label className="lot-size-option-label">
```

Comment:
To complete the association here, you want a `for={lotSize}` to attach the label to the input. We're managing “checkedness” in a different way than the browser does, so this is isn't directly crucial, but doing this the “correct” way may help screen readers as well (less sure on this point).

Reply:
Yes! Will add it in later. I also realized I forgot about adding this last night lol.

Reply:
Added in 70e9a7b